src/gcm_request.erl

```diff
@@ -66,26 +69,26 @@ send_from_project({ProjectId, Auth, RegIds, Message}, {_Key, ErrorFun}) ->
 
                 case json_post_request(Url, Headers, Body) of
                     {ok, Json} ->
-                        lager:info("FCM Project push sent: ~p~n", [Json]),
+                        lager:info([{sender_id, SenderId}, {account_token, ReceiverId}, {resource_token, ResourceId}],"FCM Project push sent: ~p~n", [Json]),
```

Comment:
maybe sender_id & receiver_id?

Reply:
I remember that we may use the existing filed to record the log. So I changed the receiver_id to account_token